src/vg/civcraft/mc/citadel/command/commands/Fortification.java

```diff
@@ -62,12 +62,12 @@ public boolean execute(CommandSender sender, String[] args) {
 		}
 		
 		PlayerType type = g.getPlayerType(uuid);
-		if (type == null){
+		if (!p.hasPermission("citadel.admin") && !p.isOp() && type == null){
 			p.sendMessage(ChatColor.RED + "You are not on this group.");
 			return true;
 		}
 		GroupPermission gPerm = gm.getPermissionforGroup(g);
-		if (!gPerm.isAccessible(type, PermissionType.BLOCKS)){
+		if (!p.hasPermission("citadel.admin") && !p.isOp() && !gPerm.isAccessible(type, PermissionType.BLOCKS)){
```

Comment:
can you change this to !(p.hasPermission("citadel.admin") || p.isOp()) && !gPerm.isAccessible(type, PermissionType.BLOCKS)